ports/foonathan-lexy/vcpkg.json

```
@@ -1,10 +1,10 @@
 {
   "name": "foonathan-lexy",
-  "version": "2022.12.1",
+  "version-string": "2025.05.0",
   "description": "C++ parsing DSL",
   "homepage": "https://github.com/foonathan/lexy",
   "license": "BSL-1.0",
-  "supports": "x64 | arm64",
+  "supports": "x86 | x64 | arm64",
```

If it supports x86, can't it do arm32?
```suggestion
```
or otherwise
```suggestion
  "supports": "!arm32",
```